### DISCUSSION TITLE
running bacalhau serve on my home machine

### DISCUSSION THREAD

**Discussion by fakela** [bacalhau-project/bacalhau#1690] [Q&A] [answered]
Can I run bacalhau serve on my home machine? What are the requirements?

**Comment by enricorotundo** [comment, marked as answer]
You can run `bacalhau serve` on any machine that fits the prerequisites listed in [the official docs](https://docs.bacalhau.org/running-node/quick-start). Note that while the walkthrough in the docs has been tested only on Ubuntu 22, bacalhau is being developed on Linux/macOS environments and therefore should work fine there as well. However, [Windows hosts are supported with limitations](https://docs.bacalhau.org/running-node/windows-support/).
